From 96946454574521d39558a2d2896b8d334b4e3a25 Mon Sep 17 00:00:00 2001
From: AdarshGrewal <adarshgrewal@gmail.com>
Date: Sun, 7 Jun 2026 15:59:45 +0530
Subject: [PATCH 3/5] Stub out call to jcodec MP4 post-processing

* this seems to be only for MP4 cover tag
---

diff --git a/smali_classes2/Pz/e.smali b/smali_classes2/Pz/e.smali
--- a/smali_classes2/Pz/e.smali
+++ b/smali_classes2/Pz/e.smali
@@ -12,6 +12,8 @@
         }
     .end annotation
 
+    return-void
+
     :try_start_0
     new-instance v0, LKz/a;
 
-- 
2.43.0
